app/serializers/export/own_post_serializer.rb

```diff
@@ -0,0 +1,33 @@
+module Export
+  # This is a serializer for the user's own posts
+  class OwnPostSerializer < FederationEntitySerializer
+    # Only for public posts.
+    # Includes URIs of pods which must be notified on the post updates.
+    # Must always include local pod URI since we will want all the updates on the post if user migrates.
+    has_many :subscribed_pods_uris
+
+    # Only for private posts.
+    # Includes diaspora* IDs of people who must be notified on post updates.
+    has_many :subscribed_users_ids
+
+    # Normally accepts Post as an object.
+    def initialize(*)
+      super
+      self.except = [excluded_subscription_key]
+    end
+
+    private
+
+    def subscribed_pods_uris
+      object.subscribed_pods_uris.push(AppConfig.pod_uri.to_s)
+    end
+
+    def subscribed_users_ids
+      object.subscribers.map(&:diaspora_handle)
```

Comment:
Yes, only the diaspora ID is used here. For the contacts the aspect names are exported [here](https://github.com/cmrd-senya/diaspora/blob/2a97a49a6f0c4a6af989deb006ae77c9347d72b9/app/serializers/export/contact_serializer.rb#L20), so that are two different things.